BrowserKit/Sources/ToolbarKit/LocationView.swift

```diff
@@ -70,19 +75,31 @@ class LocationView: UIView, UITextFieldDelegate, ThemeApplicable {
         button.addTarget(self, action: #selector(self.clearURLText), for: .touchUpInside)
     }
 
-    private lazy var urlTextField: UITextField = .build { urlTextField in
+    private lazy var searchEngineImageView: UIImageView = .build { imageView in
+        imageView.layer.cornerRadius = UX.cornerRadius
+    }
+
+    private lazy var searchEngineButton: UIButton = .build()
+
+    private lazy var urlTextField: UITextField = .build { [self] urlTextField in
         urlTextField.accessibilityIdentifier = "url"
+        urlTextField.placeholder = "Search or enter address"
```

Comment:
This string should be localizable and should be passed in through the configure method.